src/SamplesApp/SamplesApp.UITests/BenchmarkDotNetTests.cs

```diff
@@ -59,7 +59,7 @@ bool IsTestExecutionDone()
 
 			_app.WaitForElement(runButton);
 
-			TakeScreenshot("Begin", ignoreInSnapshotCompare: true);
+			using var _1 = TakeScreenshot("Begin", ignoreInSnapshotCompare: true);
```

Comment:
It looks like the [`ScreenshotInfo` destructor](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/ScreenshotInfo.cs#L32) is releasing the Bitmap resource as it should. Is it actually necessary to create a dummy variable every time TakeScreenshot() is called?

Reply:
I don't think this PR is required anymore, should we close it @carldebilly ?